src/app/prevention/strides-against/strides-against.component.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { LanguageService } from '../../services/language.service';
+import { SharedModule } from '../../shared.module';
+import { HttpClient, HttpHandler } from '@angular/common/http';
+import { ToastrService, ToastrModule } from 'ngx-toastr';
+import { APIService } from '../../services/api.service';
+import { DashboardService } from '../../services/dashboard.service';
+import { SharedDataService } from '../../services/shared.data.service';
+import { RouterTestingModule } from '@angular/router/testing';
+import { MaterialModule } from '../../material.module';
+
+import { StridesAgainstComponent } from './strides-against.component';
+
+describe('StridesAgainstComponent', () => {
+  let component: StridesAgainstComponent;
+  let fixture: ComponentFixture<StridesAgainstComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [StridesAgainstComponent],
+      imports: [RouterTestingModule, ToastrModule.forRoot(), SharedModule, MaterialModule],
+      providers: [LanguageService, HttpClient, HttpHandler, ToastrService, APIService, DashboardService, SharedDataService],
+
+    })
+      .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(StridesAgainstComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});

```

Comment:
new line.
Can there be more tests here?

Reply:
For games, till now I was using Raphael. It is a well-tested library. Are more tests required?

Reply:
You may ignore the library's behaviour if it is a well tested library but you do need to add tests to check if the dialogs shown are the right ones. So basically, you need to check the behaviour of the game. For example, an incorrect answer should take the man back to the first position/index.